### ISSUE TITLE
Windows 1809: flannel failure after windows update -- failed to hot attach bridge HNSEndpoint cbr0_ep

### ISSUE THREAD

**Issue by davemeier** [flannel-io/flannel#1112]
## Expected Behavior
flannel should run successfully on Windows 1809.

## Current Behavior
flannel fails with error:
I0313 18:02:19.778526    3232 hostgw_windows.go:183] Created HNSNetwork cbr0
I0313 18:02:19.783291    3232 hostgw_windows.go:212] Attempting to create bridge HNSEndpoint &{Id: Name:cbr0_ep VirtualNetwork:CEA05C9C-5151-46DC-A7D2-8B2DD94DC096 VirtualNetworkName: Policies:[] MacAddress: IPAddress:10.244.2.2 DNSSuffix: DNSServerList: GatewayAddress: EnableInternalDNS:false DisableICC:false PrefixLength:0 IsRemoteEndpoint:false Namespace:<nil>}
I0313 18:02:19.804296    3232 hostgw_windows.go:217] Created bridge HNSEndpoint cbr0_ep
I0313 18:02:19.805433    3232 hostgw_windows.go:221] Waiting to attach bridge endpoint cbr0_ep to host
E0313 18:02:25.580117    3232 main.go:289] Error registering network: failed to hot attach bridge HNSEndpoint cbr0_ep to host compartment: hnsCall failed in Win32: The wrong diskette is in the drive.
Insert %2 (Volume Serial Number: %3) into drive %1. (0x22)
I0313 18:02:25.594754    3232 main.go:366] Stopping shutdownHandler...

## Steps to Reproduce (for bugs)
1. K8S processes and flannel were running on the Windows 1809 node and all was working. Running these with the start.ps1 script from SDN.
2. Windows did an auto-update and restarted during the night.
3. Logged in this morning and ran the start.ps1 script.
4. Flannel error above occurred.
5. Tried restarting again, but same error happens every time I run start.ps1 on this node.

## Your Environment
* Flannel version: **v0.11.0**
* Backend used (e.g. vxlan or udp): **host_gw**
* Kubernetes version (if used): **v1.13.2**
* Operating System and version: **Windows 1809**

**Comment by davemeier** [comment]
Ran collectlogs.ps1 and attached here:

[app27_cbr0_wrong_diskette_logs_2.zip](https://github.com/coreos/flannel/files/2963165/app27_cbr0_wrong_diskette_logs_2.zip)

**Comment by ddebrecenijr** [comment]
I had similar issues, followed [Microsoft Docs](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/common-problems.md#common-networking-errors) and it fixed my issue.

**Comment by davemeier** [comment]
> 
> 
> I had similar issues, followed [Microsoft Docs](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/common-problems.md#common-networking-errors) and it fixed my issue.

Thanks @ddebrecenijr - I got some help from a Developer at Microsoft, and this is very similar to what I did to fix it. I did not delete HNS.data though. I also did not stop the hns service before deleting the cbr0 network with hnsdiag. I did have to delete the cbr0 NIC from the registry as described in the doc you linked. After that and rebooting the machine, it fixed the problem.
